src/FirebirdSql.Data.FirebirdClient/Common/DbField.cs

```diff
@@ -130,6 +130,7 @@ public string Alias
 	public Charset Charset
 	{
 		get { return _charset; }
+		set { _charset = value; }
```

Comment:
`Charset` is handled via `SubType` and these need (should) stay in sync. Similarly for `CharCount`.